lib/notifications/client/response_costdetails.rb

```diff
@@ -0,0 +1,35 @@
+require 'time'
+
+module Notifications
+  class Client
+    class CostDetails
```

Comment:
This class isn't used - cost details in the response when you get a notification is returned as a hash. I think it's fine that we return a hash since the nested template information that gets returned is also a hash, so it keeps things consistent. But it means this file can be deleted.

Reply:
removed :
https://github.com/alphagov/notifications-ruby-client/pull/143/commits/2dd92653b2207ec2a4daf518993900d17a8ab8fe